manifests/e/Evoluent/EvoluentMouseManager/6.0.9.3/Evoluent.EvoluentMouseManager.installer.yaml

```
@@ -1,30 +1,30 @@
-# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
-
-PackageIdentifier: Evoluent.EvoluentMouseManager
-PackageVersion: 6.0.9.3
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-InstallerSuccessCodes:
-- 1603
-- 3010
-InstallerSwitches:
-  Custom: REBOOTISREQUIRED=0
-FileExtensions:
-- evo
-Installers:
-- InstallerLocale: en-US
-  Architecture: x86
-  InstallerType: msi
-  Scope: machine
-  InstallerUrl: https://evoluent.com/download/EvoSetup-6.0.9.3.msi
-  InstallerSha256: 1054C4F3BEEB335A1A4E2D9DA1FD2965EF52E07B68FC4CB5545CCFBCE6C4EA58
-  UpgradeBehavior: uninstallPrevious
-  ProductCode: '{933B0FA1-2ECF-4B3F-8153-BEBD8750FF72}'
-ManifestType: installer
-ManifestVersion: 1.0.0
+# Automatically updated by the winget bot at 2021/Nov/02
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Evoluent.EvoluentMouseManager
+PackageVersion: 6.0.9.3
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSuccessCodes:
+- 1603
+- 3010
+InstallerSwitches:
+  Custom: REBOOTISREQUIRED=0

```

```suggestion
  Custom: REBOOTISREQUIRED=0 /norestart
```

Just a note that whenever the bug with 3010 gets fixed, this manifest also needs the /norestart flag